srcpkgs/kitty/template

```diff
@@ -36,7 +41,22 @@ do_install() {
 
 kitty-terminfo_package() {
 	short_desc+=" - terminfo data"
+	depends="ncurses"
 	pkg_install() {
 		vmove usr/share/terminfo
 	}
 }
+
+kitty-shell-integration_package() {
+	short_desc+=" - shell integration scripts"
+	pkg_install() {
+		vmove usr/lib/kitty/shell-integration
+	}
+}
+
+kitty-kitten_package() {
+	short_desc+=" - standalone executable"
```

Comment:
```suggestion
	short_desc+=" - kitten client"
```
[alpine](https://pkgs.alpinelinux.org/package/edge/community/aarch64/kitty-kitten) describes it as this